packages/utilities/src/utilities/StickyHeader/StickyHeader.js

```diff
@@ -114,7 +114,7 @@ class StickyHeader {
       const leadspaceSearchBar = component.shadowRoot.querySelector(
         `.${prefix}--search-container`
       );
-      this._leadspaceWithSearchBar = leadspaceSearchBar;
+      this._leadspaceSearchBar = leadspaceSearchBar;
```

Comment:
This and the following similar change was done to match the name of the property defined in this class's constructor.